.github/workflows/docs.yml

```diff
@@ -0,0 +1,42 @@
+name: Receive Dispatch and Pull Docs
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types: [docs]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  pull-docs:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v4
+
+    - name: Pull docs from rv-docs repository
+      run: | # cp -rfL docs/* ../docs fail? // cp -rf repositories/*/docs ../docs/* work but worse
+        git clone https://github.com/ReVanced/revanced-documentation rv-docs && cd rv-docs
+        git submodule update --init --recursive
```

Comment:
Can't you use an action? Would be more elegant.

Reply:
I prefer to write my first iteration of my CI with pure commands

action can be made at later iterations.